en/02-preparing-your-project-being-open-sourced.md

```diff
@@ -1,70 +1,67 @@
 
-[OSI](https://opensource.org/) is an organism that has been promoting open source software and communities for over 20 years. They have a process to review licenses. The OSI-approved licenses are the most popular ones like [MIT](https://opensource.org/licenses/MIT) or [Apache 2.0](https://opensource.org/licenses/Apache-2.0). You can find a [good comparison](https://choosealicense.com/licenses/) between licenses on [choosealicense.com](https://choosealicense.com/).
+[OSI](https://opensource.org/) est un organisme qui fait la promotion des logiciels open source et des communautés depuis plus de 20 ans. Ils ont un processus pour examiner les licences. Les licences approuvées par l'OSI sont les plus populaires comme [MIT](https://opensource.org/licenses/MIT) ou [Apache 2.0](https://opensource.org/licenses/Apache-2.0). Vous pouvez trouver une [bonne comparaison](https://choosealicense.com/licenses/) entre les licences sur [choosealicense.com](https://choosealicense.com/).
 
 ## Contributing guide
 
-A CONTRIBUTING.md file, in your open source repository or site, provides potential project contributors with a short guide to how they can help with your project or study group. It is convention to capitalize the word "contributing" as the file title, and to save it as a resource in markdown (hence the extension .md). -- [Mozilla Science Lab](https://mozillascience.github.io/working-open-workshop/contributing/)
-
-Earlier we talked about "The open source contributor funnel" in our readme, we provided documentation to help _users_ installing and using your project, we will now focus on _contributors_. The contributing guide is designed to give instructions to everybody that wants to participate to your project.
+Un fichier CONTRIBUTING.md, situé dans votre référentiel ou site open source, fournit aux contributeurs potentiels à un projet un guide expliquant comment ils peuvent vous aider dans votre projet ou votre groupe d'étude. Il est de règle de mettre le mot "contribuer" en majuscule en tant que titre du fichier et de l'enregistrer en tant que ressource dans markdown (d'où l'extension .md). - [laboratoire scientifique Mozilla](https://mozillascience.github.io/working-open-workshop/contributing/)
 
-Most of the time contributions are made by users because they encounter an issue using your project. It's nice to have instructions on how to report a bug or suggest a new feature.
+Plus tôt, nous avions parlé de "l'entonnoir des contributeurs open source" dans notre fichier Lisez-moi, nous avons fourni de la documentation pour aider les utilisateurs à installer et à utiliser votre projet. Nous allons maintenant nous concentrer sur les "contributeurs". Le guide de contribution est conçu pour donner des instructions à tous ceux qui souhaitent participer à votre projet.
```

Comment:
don't translate 'Readme'.
Make sure to keep markdown styling with italic on 'utilisateurs' and 'contributeurs'